### ISSUE TITLE
No found compiler

### ISSUE THREAD

**Issue by tattoosfour** [armory3d/armory#162]
There is no complete source code missing files I can not compile it, I've spent weeks investigating and can only compile armory_ci, I've created up a khafile.js file and it gives me krafix compiler shader error, I do not have $ 50 dollars to buy something I do not I can prove I am left with unreal and unity. If you have the solution to compile, I would be very grateful.

**Comment by luboslenco** [comment]
Hi, please drop me an [email](http://armory3d.org/community.html) to resolve this. The reason for itch.io distribution right now is to keep development going at full pace, till different ways to fund the project arise (which I will attempt to do this year, once the engine gets more mature).

**Comment by tattoosfour** [comment]
Hi, I have been doing khafile.js and I have compiled.glsl from armory_ci
that I needed to armory and I get the following error.

"Usage: krafix profile in out tempdir system"

Until I have arrived with kodeStudio u with Node in both Windows and Linux
I think it lacks files or is failure of my installation of Kha, that was my
problem that is why I deduced that files were missing to compile it.

A greeting.

2017-04-07 10:07 GMT+02:00 Lubos Lenco <notifications@github.com>:
